components/bl-list-component/README.md

```diff
@@ -0,0 +1,55 @@
+# List
+
+List is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows you to add a standard list to your application.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="643"/>
```

Comment:
Default width for thumbnail is 780px. Mark will provide production image later, but this row should be changed.